tests/find_tests/test_logging.py

```diff
@@ -4,4 +4,5 @@ def test_download_logging(find_test_client, caplog, mocked_routes_trigger_async_
     assert len(log_line) == 1
     assert log_line[0].request_type == "download"
     assert log_line[0].user_id == "test-user"
-    assert log_line[0].query_params == {"file_format": "xlsx", "email_address": "test-user@communities.gov.uk"}
+    assert log_line[0].query_params == {"file_format": "xlsx"}
+    assert "email_address" not in log_line[0].query_params
```

Comment:
question: These checks are redundant, no?

Reply:
Yes, kind of redundant, I'm just hesitant to delete tests. 
All its use at the moment is to check that in find/main/routes/download the email_address has indeed been removed from query_params:
`query_params_without_email_address = {k: v for k, v in query_params.items() if k != "email_address"}
`
Not much of a test. Shall I delete it?

Reply:
I suppose I mean that - we only need one of these two lines. Each line is testing the same thing - that `email_address` is not present.

Reply:
Sorry, I misunderstood you. I removed one of the lines.